meta-dts-distro/recipes-dts/reports/dasharo-hcl-report/dasharo-hcl-report

```diff
@@ -135,8 +135,19 @@ filename+="_$(date +'%Y_%m_%d_%H_%M_%S_%N')"
 filename="${filename// /_}"
 
 echo "Creating archive with logs..."
-tar -zcf "$filename.tar.gz" logs/*
-rm -rf logs
+if [ "$SEND_LOGS" = "true" ]; then
```

Comment:
@kotylamichal lets just remove the mac address from the logs without saving it in local copy.

Reply:
@TomaszAIR done